DigitalRain/Raindrops/PerColumnSpaceRaindropFactory.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Xna.Framework;
+
+namespace DigitalRain.Raindrops
+{
+    using Columns;
+
+    public class PerColumnSpaceRaindropFactory : IRaindropFactory
+    {
+        private static readonly Random _randomGen = new Random();
+        private readonly StandardRaindropFactoryConfig _config;
+        private readonly Dictionary<(int, int), char[]> _symbolPools;
+        private readonly Dictionary<(int, int), double> _lifespans;
+
+        public PerColumnSpaceRaindropFactory()
+        {
+            _config = DigitalRainGame.Config.standardRaindropFactory;
+
+            _symbolPools = new Dictionary<(int, int), char[]>();
+            _lifespans = new Dictionary<(int, int), double>();
+            InitializeSymbolPoolsAndLifespans(rowNumber: 7, fixedText: "HELLO WORLD!", fixedLifespan: 120000); // 2 minutes
```

Comment:
Maybe inside `GameUtilities` we could have a math function to convert 2 minutes to milliseconds?

Or am I just too smooth brained and we should keep it as is? Seems like milliseconds is the standard so having simple converting functions may help us express things more clearly.

Reply:
We should, I'm just lazy.  I haven't looked, but we might also be able to do:
```
new GameTime.Minutes(2).TotalMilliseconds()
```